src/tooltip.ts

```diff
@@ -363,7 +363,8 @@ const tooltipPlugin = ViewPlugin.fromClass(class {
 
   maybeMeasure() {
     if (this.manager.tooltips.length) {
-      if (this.view.inView) this.view.requestMeasure(this.measureReq)
+      if (!this.view.hasFocus) for (let tv of this.manager.tooltipViews) tv.dom.style.top = Outside
```

Comment:
I think we're going to have to do something with inView as well, since that will stomp over whatever this sets it.

Reply:
Both this line and the other line below that sets `top = Outside` are just moving the tooltip out of view. The only place where it's move back into view is in `requestMeasure` which we're now preventing from being called when there is no focus. So I think that should cover the cases.

Though the inView is hard to test because ours never goes out of view.

https://github.com/user-attachments/assets/66e66744-0582-4c99-a6f0-d89061981c79

Reply:
Oh yeah you are right.